build.gradle.kts

```diff
@@ -24,6 +24,8 @@ val addOpens = listOf(
     "java.base/java.nio=ALL-UNNAMED",
     "java.base/java.io=ALL-UNNAMED",
     "java.base/java.lang=ALL-UNNAMED",
+    "java.base/java.lang.ref=ALL-UNNAMED",
```

Comment:
Ran into this issue on my server when saving data files for Forge Essentials using the bundled gson library
Since Forge Essentials does not bundle gson, I think this is a fix that belongs in lwjgl3ify since it might effect other mods as well.